furms-integration-tests/src/test/java/io/imunity/furms/integration/tests/rest/project/ProjectsAllocationsIntegrationTest.java

```diff
@@ -187,25 +187,6 @@ void shouldFindProjectAllocationByProjectIdAndAllocationId() throws Exception {
 				.andExpect(jsonPath("$.amount", equalTo(2)));
 	}
 
-	@Test
-	void shouldNotFindProjectAllocationByProjectIdAndAllocationIDueToLackOfCorrectRights() throws Exception {
```

Comment:
It has removed, because now rights to this method changed from PROJECT_READ to PROJECT_LIMITED_READ, because even project user need those data for his charts.